packages/google-cloud-pubsub/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@golevelup/nestjs-google-cloud-pubsub",
+  "version": "0.0.1",
+  "description": "Google cloud pubsub wrapper for NestJS",
+  "author": "Nikita Chaikin <nikita.chaikinn@gmail.com>",
+  "homepage": "https://github.com/golevelup/nestjs/blob/master/packages/google-cloud-pubsub/README.md",
+  "license": "MIT",
+  "keywords": [
+    "NestJS",
+    "Google Cloud",
+    "PubSub",
+    "gcp"
+  ],
+  "main": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/golevelup/nestjs.git"
+  },
+  "scripts": {
+    "build": "tsc --build tsconfig.build.json",
+    "build:watch": "tsc --build tsconfig.build.json --watch",
+    "test": "jest"
+  },
+  "bugs": {
+    "url": "https://github.com/golevelup/nestjs/issues"
+  },
+  "dependencies": {
+    "@golevelup/nestjs-discovery": "workspace:^",
+    "@google-cloud/pubsub": "^5.2.0"
```

Comment:
@chaikin-nikita Does it make more sense for this to be a peerDepedency? We can always install this as part of the devDependecies but it would give more control to people if they have this lib already installed